### ISSUE TITLE
Working Without Composer

### ISSUE THREAD

**Issue by kevinkhill** [kevinkhill/lavacharts#24]
So I started to incorporate the Symfony autoloader and Carbon (from @briannesbitt) which seemed like a good idea to allow the usage of Lavacharts outside of composer. I then started thinking it might be a bad idea, to include other vendor files in the package, which would possibly double up when used with composer. I suppose I could create another repo, for a composerless install including the dependencies as well as this repo, which would be dedicated to using composer.

Basically, I don't know which would be the best route to take to include everyone. If anyone could give me ideas or guidance on the best route to take I would greatly appreciate it.

**Comment by approached** [comment]
I have create a autoload.php class:
```
function google_chart_autoload($className)
{
    $classPath = explode('\\', $className);
    if ($classPath[0] != 'Khill') {
        return;
    }
    $filePath = dirname(__FILE__) . '/src/' . implode('/', $classPath) . '.php';

    if (file_exists($filePath)) {
        require_once($filePath);
    }
}

spl_autoload_register('google_chart_autoload');
```

Than i can easy use the classe like this: $stocksTable = new Khill\Lavacharts\Configs\DataTable();

**Comment by kevinkhill** [comment]
I was looking into something similar, like how the Symfony autoloader works. I get how that would load all of the Lavacharts classes, but then would I just add a note in the instructions for a composer-less install that users have to also download and include Carbon?

**Comment by approached** [comment]
Carbon is just one file what you need to include.
Like this: require_once '/Carbon/src/Carbon/Carbon.php';

**Comment by kevinkhill** [comment]
A fix is coming soon....
